### PR TITLE
update to latest akka version (2.6)

### PR DESCRIPTION
**Pull request by diclophis**
to test updated version of akka

https://akka.io/docs/other-releases.html akka 2.5 seams to be EOL, going to akka-2.6 is ideal

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -2,7 +2,7 @@ import sbt._
 
 object Dependencies {
   // Versions
-  lazy val akkaVersion = "2.5.14"
+  lazy val akkaVersion = "2.6.15"
```
Comment (diclophis) [inline project/Dependencies.scala]: @jacksenechal I would like to pair in on seeing what this error means ... all I can dig up is this stackoverflow, but its def all greek https://stackoverflow.com/questions/24931902/unresolved-dependencies-error-in-akka

Comment (diclophis) [reply project/Dependencies.scala]:
```
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	::          UNRESOLVED DEPENDENCIES         ::
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	:: com.typesafe.akka#akka-actor_2.11;2.6.15: not found
[warn] 	:: com.typesafe.akka#akka-agent_2.11;2.6.15: not found
[warn] 	:: com.typesafe.akka#akka-slf4j_2.11;2.6.15: not found
[warn] 	:: com.typesafe.akka#akka-testkit_2.11;2.6.15: not found
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 
[warn] 	Note: Unresolved dependencies path:
[warn] 		com.typesafe.akka:akka-actor_2.11:2.6.15 (/root/current/build.sbt#L75)
[warn] 		  +- mavenlink:changestream_2.11:0.4.0-577571578c82cacb154b1853387b14cebf3d3cbe
[warn] 		com.typesafe.akka:akka-agent_2.11:2.6.15 (/root/current/build.sbt#L75)
[warn] 		  +- mavenlink:changestream_2.11:0.4.0-577571578c82cacb154b1853387b14cebf3d3cbe
[warn] 		com.typesafe.akka:akka-slf4j_2.11:2.6.15 (/root/current/build.sbt#L75)
[warn] 		  +- mavenlink:changestream_2.11:0.4.0-577571578c82cacb154b1853387b14cebf3d3cbe
[warn] 		com.typesafe.akka:akka-testkit_2.11:2.6.15 (/root/current/project/Testing.scala#L43)
[warn] 		  +- mavenlink:changestream_2.11:0.4.0-577571578c82cacb154b1853387b14cebf3d3cbe
sbt.ResolveException: unresolved dependency: com.typesafe.akka#akka-actor_2.11;2.6.15: not found
```

Comment (jacksenechal) [reply project/Dependencies.scala]: Interesting. Yes that sounds like a good idea!

**Comment by diclophis** [comment]
gonna close this, I was hoping to be able to pull to the major version but I don't think I will have time any time soon to pull this forward

**Pull request closed without merging**